app/SearchApi/SearchRequest.Adaptor/Notifier/SearchRequestNotifier.cs

```diff
@@ -0,0 +1,102 @@
+﻿using BcGov.Fams3.SearchApi.Contracts.PersonSearch;
+using BcGov.Fams3.SearchApi.Contracts.SearchRequest;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using SearchRequestAdaptor.Configuration;
+using SearchRequestAdaptor.Publisher;
+using SearchRequestAdaptor.Publisher.Models;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SearchRequestAdaptor.Notifier
+{
+    public interface ISearchRequestNotifier<T>
+    {
+        Task NotifySearchRequestEventAsync(string searchRequestKey, T searchRequestOrdered, CancellationToken cancellationToken);
+    }
+
+    public class WebHookSearchRequestNotifier : ISearchRequestNotifier<SearchRequestOrdered>
+    {
+
+        private readonly HttpClient _httpClient;
+        private readonly SearchRequestAdaptorOptions _searchRequestOptions;
+        private readonly ILogger<WebHookSearchRequestNotifier> _logger;
+        private readonly ISearchRequestEventPublisher _searchRequestEventPublisher;
+
+
+        public WebHookSearchRequestNotifier(HttpClient httpClient, IOptions<SearchRequestAdaptorOptions> searchRequestOptions, ILogger<WebHookSearchRequestNotifier> logger, ISearchRequestEventPublisher searchRequestEventPublisher)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+            _searchRequestOptions = searchRequestOptions.Value;
+            _searchRequestEventPublisher = searchRequestEventPublisher;
+        }
+
+        public async Task NotifySearchRequestEventAsync(string searchRequestKey, SearchRequestOrdered searchRequestOrdered,
+           CancellationToken cancellationToken)
+        {
+            var webHookName = "SearchRequest";
+            string eventName = "SearchRequestOrdered";
+
+            if (searchRequestOrdered == null) throw new ArgumentNullException(nameof(SearchRequestOrdered));
+
+            foreach (var webHook in _searchRequestOptions.WebHooks)
+            {
+                _logger.LogDebug(
```

Comment:
Are we really going to have mulitple endpoints ? 
If we do will we be sending the same message to them ?

Reply:
I thought that's one of the webhook idea.